### ISSUE TITLE
The 'ImageOptimizer' package did not load correctly

### ISSUE THREAD

**Issue by miguellira** [madskristensen/ImageOptimizer#23]
### Installed product versions
- Visual Studio: 2013 Ultimate
- This extension: 3.6.92
### Description

When opening a solution in Visual Studio 2013 I am presented with the dreaded `The 'ImageOptimizer' package did not load correctly.` error message.
### Steps to recreate
1. Install the VSIX
2. Open an existing solution
### Current behavior

When opening a solution in Visual Studio 2013 I am presented with the dreaded `The 'ImageOptimizer' package did not load correctly.` error message.

The ActivityLog.xml file contains the following entries:

``` xml
<entry>
    <record>770</record>
    <time>2016/08/01 14:52:36.572</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>CreateInstance failed for package [ImageOptimizerPackage]</description>
    <guid>{BF95754F-93D3-42FF-BFE3-E05D23188B08}</guid>
    <hr>80070002</hr>
    <errorinfo>Could not load file or assembly 'Microsoft.VisualStudio.Shell.11.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.</errorinfo>
  </entry>
  <entry>
    <record>771</record>
    <time>2016/08/01 14:52:36.572</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>End package load [ImageOptimizerPackage]</description>
    <guid>{BF95754F-93D3-42FF-BFE3-E05D23188B08}</guid>
    <hr>80004005 - E_FAIL</hr>
    <errorinfo>Could not load file or assembly 'Microsoft.VisualStudio.Shell.11.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.</errorinfo>
  </entry>
```
### Expected behavior

Solution opens and package loads properly

**Comment by miguellira** [comment]
Looks like it's the same issue I ran into with the AddAnyFile extension (https://github.com/madskristensen/AddAnyFile/issues/30)

**Comment by madskristensen** [comment]
This should now be fixed. Please help verify the fix by installing the [CI build](http://vsixgallery.com/extension/bf95754f-93d3-42ff-bfe3-e05d23188b08/)

**Comment by madskristensen** [comment]
Yes, it was the same issue as with the Add Any File extension

**Comment by miguellira** [comment]
Confirmed. Fixed in version 3.6.93.
